docs/source/2019Barry.rst

```diff
@@ -72,6 +72,31 @@ Model Configuration and Datasets
    
       crontab Barry.crontab  
 
+  .. group-tab:: SRW.v1.0
+
+    The UFS Short-Range Weather (SRW) Application (App) is used to prepare initial conditions, compile and run the UFS model, and post process the raw model outputs. Two model configuration compsets (``GFSv15p2`` and ``RRFSv1alpha``) are tested using the :emphasis:`C768` (~13km) spatial resolution with 64 vertical levels (default).
+
+    The case runs are initialized at 00z Jul 12, 2019 with 90 hours forecasting. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, forecast length and cycle hour are listed below.
+
+    .. code-block:: bash
+ 
+      FCST_LEN_HRS="90"
+      LBC_SPEC_INTVL_HRS="3"
+      DATE_FIRST_CYCL="20190712"
+      DATE_LAST_CYCL="20190712"
+      CYCL_HRS=( "00" )
+
+    Initial condition (IC) files are created from GFS operational dataset in NEMSIO format. The `Stand-alone Geophysical Fluid Dynamics Laboratory (GFDL) Vortex Tracker <https://dtcenter.org/community-code/gfdl-vortex-tracker>`_ is a tool to estimate hurricane tracks and intensities. The `Best Track dataset <https://www.nhc.noaa.gov/data/#hurdat>`_ provides the ‘truth’ data for hurricane evolution.
+
+    .. container:: sphx-glr-footer
+       :class: sphx-glr-footer-example
+
+
+      .. container:: sphx-glr-download sphx-glr-download-python
+
+        :download:`Download initial condition files: 2019071200.gfs.nemsio.tar.gz  <https://ufs-case-studies.s3.amazonaws.com/2019071200.gfs.nemsio.tar.gz>`
```

Comment:
We will need to ask Dom to upload the IC files for the SRW app to the cloud next week.

Reply:
Please provide me with a location of all files and I will upload them. Preferably on Cheyenne, but any NOAA platform will do.

Reply:
Does it make sense to have only one download link entry that includes both SRW's IC and BC files? like `Download initial condition and boundary condition files: 2019071200.gfs.nemsio.tar.gz`.

Reply:
The IC tar file only contains the files for initial conditions which is consistent with global model, and the BC tar file contains files for boundary conditions only needed by LAM. So it is better to separate them as Dom suggested.

Reply:
I see. Thank you @panll for the explanation!